trillian/examples/deployment/docker/ctfe/Dockerfile

```diff
@@ -1,15 +1,20 @@
 FROM golang:buster as build
 
-ADD . /go/src/github.com/google/certificate-transparency-go
-WORKDIR /go/src/github.com/google/certificate-transparency-go
-
 ARG GOFLAGS=""
 ENV GOFLAGS=$GOFLAGS
 ENV GO111MODULE=on
-RUN go get ./trillian/ctfe/ct_server
+
+WORKDIR /build
```

Comment:
no-op comment: I double-checked that`WORKDIR` will create the dir if it does not exist.